plugin/plugin_settings.py

```diff
@@ -37,9 +37,12 @@ class Settings:
     # refer to Preferences.sublime-settings for usage explanation
     NAMES_ENUM = [
         "autocomplete_all",
+        "c_flags",
         "clang_binary",
         "cmake_flags_priority",
         "cmake_prefix_paths",
+        "common_flags",
```

Comment:
It's not a problem specifically with this pull request but there is a fundamental problem with reading settings from the project file. As there is only one instance of `Settings`, the project overrides will be read only from the first project that is opened. If having a couple projects open, or switching between them, all will share settings of the first project... I can create separate issue for that if you prefer.